### PR TITLE
fix download issues

### PR DESCRIPTION
**Pull request by hsj576**
This PR aims to fix the download issues.

### CODE REVIEW

**Comment by hsj576** [comment]
@MooreZheng

File: download.html [hunk 1]
```diff
@@ -152,7 +152,7 @@ <h3><b>Instructions</b></h3>
                     <span class="anchor" id="link"></span>
                     <h3><b>Download Links</b></h3>
                     <p> 
-                        Download link:  <a href="https://www.kaggle.com/datasets/hsj576/cloud-robotics" target="_blank">Cloud-Robotics</a></br>
+                        The dataset is currently in the testing phase. If you want to get the beta version of the dataset, please email sjhu21@m.fudan.edu.cn.
```
Comment (MooreZheng) [inline download.html]: Polished version: "This dataset belongs to the perception module of the cloud-robotic benchmarking project which is in its beta stage and will be available shortly. Before the official release of the whole project, one can request a copy of this dataset by emailing Mr. Shijing via sjhu21@m.fudan.edu.cn."

**Comment by MooreZheng** [comment]
> @MooreZheng

Hi @hsj576, a revised version is as above.

**Comment by hsj576** [comment]
> > @MooreZheng
> 
> Hi @hsj576, a revised version is as above.

Ok, I have revised it.

**Comment by MooreZheng** [comment]
As for the dataset license, Apache 2.0  may not be the best fit. A new issue would be raised to take care the whole benchmark.

**Pull request merged**